plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -155,6 +160,27 @@ public void testIllegalFlushHiveMetastoreCacheProcedureCalls()
                 .hasMessage("Parameters partition_column and partition_value should have same length");
     }
 
+    @Test
+    public void testPartitionAppend()
+    {
+        verify(nodeCount > 1, "this test requires a multinode query runner");
```

Comment:
Use `getQueryRunner().getNodeCount()`. More resilient to future refactors and makes it explicit what we want to validate.

Reply:
Applied